pkg/lib/authn/sso/azureadv2.go

```diff
@@ -149,6 +150,24 @@ func (f *Azureadv2Impl) OpenIDConnectGetAuthInfo(r OAuthAuthorizationResponse, p
 	return
 }
 
+func (f *Azureadv2Impl) GetPrompt(prompt []string) []string {
+	// Azureadv2 only support single value for prompt
+	// the first supporting value in the list will be used
```

Comment:
Azureadv2 only supports single value for prompt, and I will pick the first supporting value from the list. What do you think?

Reply:
This is the best we can do. Maybe we should document the order MAY matter in the SDK documentation.